app/src/main/res/values/dimens.xml

```diff
@@ -20,18 +20,34 @@
     <!-- Per Material Design specs: -->
     <!-- https://material.io/design/components/buttons-floating-action-button.html#specs -->
     <dimen name="fab_margin">16dp</dimen>
+    <dimen name="fab_corner_radius">30dp</dimen>
```

Comment:
Advise splitting shape related resources into a separate `shape.xml` file.

Reply:
Moved FAB and add button corner radius to shape.xml

Reply:
Resolved in #477